DI-1299/compare_b37_and_b38.py

```
@@ -0,0 +1,481 @@
+        b37_df,
+        b38_df,
+        on="sample_id",
+        suffixes=("_GRCh37", "_GRCh38"),
+    )
+
+    # Reorder df so sample_id column is first + remove whitespace from headers
+    columns_to_sort = [
+        col for col in all_results.columns if col != "sample_id"
+    ]
+    reordered_df = all_results.reindex(
+        ["sample_id"] + sorted(columns_to_sort), axis=1
```

Ruff (0.8.2)

Consider `["sample_id", *sorted(columns_to_sort)]` instead of concatenation

Replace with `["sample_id", *sorted(columns_to_sort)]`

(RUF005)